### DISCUSSION TITLE
Is it possible to quickly choose either --theirs or --ours option during merge conflict?

### DISCUSSION THREAD

**Discussion by muralivnv** [jesseduffield/lazygit#1608] [Q&A]
During a merge conflict, when the file is too big, going through each hunk and selecting all the changes from ours/theirs is tedious. So, is there an option in lazygit to choose `--ours` or `--theirs`? 

If not, is there any other way to do this without using `custom command`?

**Comment by arxhive** [comment]
Only custom commands.
You can set context `files` or `global` depends on your preferences.

```
  - key: "t"
    command: "git checkout --theirs {{ .SelectedFile.Name }}"
    context: "global"
    loadingText: "take theirs"
  - key: "o"
    command: "git checkout --ours {{ .SelectedFile.Name }}"
    context: "global"
    loadingText: "take ours"
```

**Reply by stefanhaller** [reply]
Note however that for the "ours" case this is not the same as using lazygit's merge view to manually pick the "ours" hunk for every conflict. The difference is that `git checkout --ours <filename>` picks "ours" for _every_ change, even those that didn't conflict, whereas the manual method only picks "ours" for conflicting hunks, but "theirs" for all non-conflicting hunks. Since I never had a need to bulk-pick "ours", I can't tell which of the two is more likely to be what you want in a given scenario.

I do sometimes have a need to bulk-pick "theirs" though. In this case I usually use lazygit's merge view to do this; since lazygit remembers the ours/theirs choice from one hunk to the next, and auto-jumps to the next hunk, it's a simple matter of pressing space repeatedly, which I find convenient enough even when there are lots of conflicts.
